packages/cli/src/databases/migrations/common/1720101653148-AddConstraintToExecutionMetadata.ts

```diff
@@ -0,0 +1,112 @@
+import type { MigrationContext, ReversibleMigration } from '@db/types';
+import { nanoid } from 'nanoid';
+
+export class AddConstraintToExecutionMetadata1720101653148 implements ReversibleMigration {
```

Comment:
I'd like us to rewrite this migration eventually to be able to use the utility methods on `queryRunner`, to make this file less verbose. But that is not a blocker.

This is likely not an issue for `execution_metadata`, but in general, we should avoid going the explicit temporary table route because there is always a chance that the migration forgets to disable foreign key constraints, and delete-cascade causes data loss. This has happened twice already in the past with some other tables.

Reply:
🙀 
👍🏾